.circleci/config.yml

```diff
@@ -139,7 +139,7 @@ jobs:
             exit_code=0
             for pkg in $(go list ./...)
             do
-              if ! gotestsum --no-summary=all --jsonfile=jsonfile-${pkg////-} -- $pkg -p 1 -timeout 20m -failfast \
+              if ! gotestsum --no-summary=all --jsonfile=jsonfile-${pkg////-} -- $pkg -p 1 -timeout 30m -failfast \
```

Comment:
@ashwin-venkatesh bumping this up because we've run into an issue where cleanup in the test framework takes a long time to delete namespaces (~1min or more each) so the test does not complete within 20 minutes.
Have confirmed this behaviour in connect_inject_namespaces tests as well and will chase that down in the future.